viral/offline_sequence_detection.py

```
@@ -0,0 +1,1148 @@
+            place_threshold=place_threshold,
+        )
+
+    place_cells = spks[pcs_mask, :]
+
+    get_cache_path = lambda variable_name: (
+        SERVER_PATH
+        / "viral_caches"
+        / "sequence_detection"
+        / variable_name
+        / f"{session.mouse_name}_{session.date}_{"online" if bayesian_config.online else "offline"}_{"en_bloc" if bayesian_config.en_bloc else "event_by_event"}.npz"
+    )
```

Ruff (0.14.5)

Do not assign a `lambda` expression, use a `def`

Rewrite `get_cache_path` as a `def`

(E731)